src/lib/types/model.ts

```diff
@@ -342,7 +341,7 @@ export interface IMetricCounts {
 export interface IMetricsBucket {
     start: Date;
     stop: Date;
-    toggles: IMetricCounts;
+    toggles: { [key: string]: IMetricCounts };
```

Comment:
This type was not used and it was not correct. If we look at the schema, toggles is an object with each key representing the toggle name and the value being the metrics count